db/data/20210831150141_add_bursaries.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddBursaries < ActiveRecord::Migration[6.1]
```

Comment:
I feel like we don't have a great process for running the seeds in production. This code is the same as `db/data/20210702132811_create_bursaries.rb`.

Are we going to include an identical data migration each time we need to add another bursary (or change the seeds) - if so, there's already another PR that needs to include this: https://github.com/DFE-Digital/register-trainee-teachers/pull/1362 . Maybe we could extract the code rather than copypasta and call it in this migration?

Or do we run the seeds manually sometimes? cc @gpeng 

Side note - I'm not sure that the data migrations even run in prod on deployment https://trello.com/c/dIEIo0C1/2470-data-migrations-arent-running

Apologies - this is a bit of a brain dump 😂

Reply:
Discussed on slack https://ukgovernmentdfe.slack.com/archives/CGVNJ2Q2F/p1630582099067400